blocks/toc/toc.js

```diff
@@ -52,25 +52,18 @@ function buildTOCSide(ul, block) {
 
 function buildTOCTop(ul, block) {
   block.parentElement.classList.add('flat');
-  let others = false;
-  let tocIndex = 0;
-  [...document.querySelector('main').children].forEach((section, i) => {
-    if (others) {
-      const liItem = ul.querySelectorAll('li')[i - tocIndex - 1];
-      if (liItem) {
-        const liVar = liItem.innerText.toLowerCase();
-        section.id = liVar;
-        const aLink = document.createElement('a');
-        aLink.href = `#${liVar}`;
-        aLink.innerText = liItem.innerText;
-        liItem.innerText = '';
-        liItem.append(aLink);
-      }
-    }
-    if (section.classList.contains('toc-container')) {
-      others = true;
-      tocIndex = i;
-    }
+
+  [...ul.querySelectorAll('li')].forEach((li) => {
```

Comment:
Could this also be implemented for the side TOC as well to maintain consistency?

Reply:
@dnbute Unfortunately I don't have this needs for current page. If you have we can do it, or we can do it in the future.